common/src/main/java/org/opennms/integration/api/v1/timeseries/immutables/ImmutableTag.java

```diff
@@ -43,7 +43,7 @@
 
     public ImmutableTag(String key, String value) {
         this.key = key;
-        this.value = Objects.requireNonNull(value, key);
+        this.value = Objects.requireNonNull(value);
```

Comment:
this was done here: https://github.com/OpenNMS/opennms-integration-api/commit/91c26ade6a5c7bbfa45f90b1663b011a14ce0812#diff-334290431a20a6a0222b0a3446299891078eab2b92c7d8aa77535d3d10a86b5b
calling `String.format` is too expensive, using the `key` as is cheap

I agree that it does look out of place though, should we put it back and add a comment?

Reply:
yeah, let's do do that. It looks strange.